packages/dashboard-v2/src/components/NavBar/NavBar.js

```diff
@@ -1,4 +1,4 @@
-import { Link } from "gatsby";
+import Link from "gatsby-link";
```

Comment:
https://www.npmjs.com/package/gatsby-link

> All components and utility functions from this package are now exported from [gatsby](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby) package. You should not import anything from this package directly.

Reply:
Changed in 5123a23e -- making it work requires a Storybook addon.